src/main/java/by/exadel/internship/controller/FormController.java

```diff
@@ -1,25 +1,23 @@
 package by.exadel.internship.controller;
 
 import by.exadel.internship.dto.formDTO.FormRegisterDTO;
+import by.exadel.internship.service.FormService;
 import io.swagger.annotations.Api;
 import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.ArrayList;
-import java.util.List;
-
 @RestController
 @RequestMapping("/forms")
 @Api(tags = "Form endpoints")
 public class FormController {
 
-    static private List<FormRegisterDTO> formList = new ArrayList<>();
+    @Autowired
+    public FormService formService;
 
     @PostMapping
     @ApiOperation("Add new form")
-    public FormRegisterDTO addNewForm(@RequestBody FormRegisterDTO form) {
-        formList.add(form);
-        return form;
+    public void addNewForm(@RequestBody FormRegisterDTO form) {
+        formService.saveForm(form);
```

Comment:
Should this endpoint return ResponseEntity?

Reply:
il do it in next commit